content/docs/rfcs/10/README.md

```diff
@@ -133,6 +133,12 @@ For an experimental privacy-preserving economic spam protection mechanism, see [
 
 See [23/WAKU2-TOPICS](/spec/23) for more information about recommended topic usage.
 
+# Transport Protocol Specification
```

Comment:
```suggestion
# Transports
```

Reply:
done in ea014867f9062d531dd9f1772c99e4b2058f5f80